lib/utils.js

```diff
@@ -239,12 +247,14 @@ function buildVersionUrl(spec) {
       return `${buildRegistryUrl(spec)}/${spec.revision}`
     case 'rubygems':
       return `${buildRegistryUrl(spec)}/versions/${spec.revision}`
+    case 'cocoapods':
+      return `https://cocoapods.org/pods/${spec.name}`
     default:
       return null
   }
 }
 
-async function buildDownloadUrl(spec) {
+function buildDownloadUrl(spec) {
```

Comment:
might be good to add a comment here that this will build URLs for any protocol with URLs that can be predicted from the identity of the component